### PR TITLE
add avatar picture to TaskCard

### LINKED ISSUE #770: Avatar icon in task card
### Describe the feature

The avatar icon should be added to the task card as visible in the mock https://github.com/helpwave/design/issues/8#issuecomment-1763477668.

### Why is this feature needed?

To give the user a sense if a task is assigned or not.

### Additional context

Until https://github.com/helpwave/services/issues/476 is finished, you can use https://source.boringavatars.com/marble/128/ for a reference icon and open a draft pr.

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #770

### CODE REVIEW

File: tasks/components/cards/TaskCard.tsx [hunk 1]
```diff
@@ -15,23 +32,29 @@ export type TaskCardProps = CardProps & {
...
   onTileClick = () => undefined
-}: TaskCardProps) => {
+}: PropsWithLanguage<TaskCardTranslation, TaskCardProps>) => {
+  const translation = useTranslation(language, defaultTaskCardTranslations)
   const progress = task.subtasks.length === 0 ? 1 : task.subtasks.filter(value => value.isDone).length / task.subtasks.length
   const isOverDue = task.dueDate && task.dueDate < new Date() && task.status !== TaskStatus.TASK_STATUS_DONE
 
+  // TODO replace by user avatar
+  const tempURL = 'https://source.boringavatars.com/marble/128/'
+  const hasAssignee = !!task.assignee && task.assignee !== '00000000-0000-0000-0000-000000000000'
```
Comment (use-to) [inline tasks/components/cards/TaskCard.tsx:46]:
```suggestion
  // TODO replace by user avatar
  // Fetch for specific assignee https://github.com/helpwave/services/blob/c1c88ebaad136aef92954bf774b1e89c6a10d97f/services/user-svc/internal/models/user_models.go#L7
  const tempURL = 'https://source.boringavatars.com/marble/128/'
  const hasAssignee = !!task.assignee && task.assignee !== '00000000-0000-0000-0000-000000000000'
```

**Comment by use-to** [comment]
Depends on https://github.com/helpwave/services/pull/484

File: tasks/mutations/user_mutations.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+const userQueryKey = 'user'
+
+export const useUserQuery = (userId: string|undefined) => {
+  const enabled = !!userId && userId !== '00000000-0000-0000-0000-000000000000'
+  return useQuery({
+    queryKey: [userQueryKey, userId],
+    enabled,
+    staleTime: 5 * 60 * 1000, // 5 minutes
+    cacheTime: 30 * 60 * 1000, // 30 minutes
```
Comment (use-to) [inline tasks/mutations/user_mutations.ts:13]: Use config or constant variables

**Review by use-to — COMMENTED** [review]
Looks good. Approval guaranteed atm...
Please update the package version on services increment.

**Review by use-to — DISMISSED** [review]
image
Consider the comment above.

File: tasks/mutations/user_mutations.ts (hunk 2 above)
Comment (MaxSchaefer) [reply tasks/mutations/user_mutations.ts:13]: These values will most likely not be reused and are specific to this query.

**Review by use-to — APPROVED** [review]

**Review by florian-sabonchi — APPROVED** [review]

**Pull request merged**